praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/exception/GlobalExceptionHandler.java

```diff
@@ -43,4 +44,11 @@ public String IllegalArgumentException(IllegalArgumentException ex) {
         return ex.getMessage();
     }
 
+    // Is thrown when the Database encounters an error
+    @ExceptionHandler(PSQLException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ResponseBody
+    public String PsqlException(PSQLException ex) {
+        return ex.getMessage();
+    }
```

Comment:
I don't think we should map the PSQLException to HTTP400 by default, because there are various reasons for this Exception eg. a corrupt db schema, connection failure or malformed sql statements, just to mention a few. Mapping all of them to 400 would be a bad design in my opinion, as we would blame the user for issues he can not change and might only be changed by us with a new deployment